after/ftplugin/qf.vim

```diff
@@ -127,6 +127,10 @@ command! -buffer ListLists call qf#namedlist#ListLists()
 " remove given lists or all
 command! -buffer -nargs=* -bang -complete=customlist,qf#namedlist#CompleteList RemoveList call qf#namedlist#RemoveList(expand("<bang>") == "!" ? 1 : 0, <q-args>)
 
+" navigate between older and newer lists
+nnoremap <silent> <buffer> <Left> :call qf#history#Older()<CR>
```

Comment:
Like with my `(}` mappings below I'm not comfortable with having `<Right><Left>` hardcoded like that. But I don't have a good solutions at the moment.

It would be easy to turn them into `<Plug>` mappings but customizing them would be too involved. Or I could use a `g:variable` but that would be incoherent with the other mappings.

Reply:
I added `<Plug>` mappings so you can consider how they'd look. I'm comfortable with removing the default `qf`-buffer-local mappings and making this feature optional (available for users to map themselves) if you'd prefer that approach.